COMPATIBILITY.md

```diff
@@ -87,7 +91,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `ec2`: known to work as is
```

Comment:
The version specifications above do not provide enough information to me. Does it mean that only one version is supported? Or do we support all further versions?